BitwardenShared/UI/Platform/Settings/Settings/AutoFill/PasswordAutoFill/PasswordAutoFillView.swift

```diff
@@ -5,9 +5,97 @@ import SwiftUI
 /// A view that shows the instructions for enabling password autofill.
 ///
 struct PasswordAutoFillView: View {
+    // MARK: Properties
+
+    /// The store used to render the view.
+    @ObservedObject var store: Store<
+        PasswordAutoFillState,
+        Void,
+        PasswordAutoFillEffect
+    >
+
+    /// An object used to determine the current color scheme.
+    @Environment(\.colorScheme) var colorScheme
+
+    /// An object used to open urls from this view.
+    @Environment(\.openURL) private var openURL
+
     // MARK: View
 
     var body: some View {
+        Group {
+            if store.state.nativeCreateAccountFeatureFlag {
+                contentView
+            } else {
+                legacyContentView
+            }
+        }
+        .navigationBar(
+            title: store.state.navigationBarTitle,
+            titleDisplayMode: .inline
+        )
+        .task {
+            await store.perform(.appeared)
+        }
+    }
+
+    // MARK: Private Views
+
+    /// The content view.
+    private var contentView: some View {
+        VStack(spacing: 0) {
+            ZStack {
+                gifViewPlaceholder
+                    .frame(width: 230, height: 278)
+                    .background(.red)
+                    .padding(.top, 32)
+
+                gifView
+                    .frame(width: 230, height: 280)
+                    .padding(.top, 32)
+            }
+
+            Text(Localizations.turnOnAutoFill)
+                .styleGuide(.title2, weight: .bold)
+                .padding(.top, 32)
+
+            Text(Localizations.useAutoFillToLogIntoYourAccountsWithASingleTap)
+                .styleGuide(.body)
+                .multilineTextAlignment(.center)
+                .padding([.top, .horizontal], 16)
+
+            autofillInstructions
+                .padding(.top, 32)
+
+            Text(
+                LocalizedStringKey(
+                    Localizations.needHelpCheckOutAutofillHelp(
+                        ExternalLinksConstants.autofillHelp)
+                )
+            )
+            .styleGuide(.subheadline)
+            .tint(Asset.Colors.textInteraction.swiftUIColor)
+            .padding(.top, 32)
+
+            Button(Localizations.continue) {
+                openURL(ExternalLinksConstants.passwordOptions)
```

Comment:
ℹ️ On iOS 17+ there's a more direct way to go there if I'm understanding correctly where this should be headed with the [ASSettingsHelper](https://developer.apple.com/documentation/authenticationservices/assettingshelper/opencredentialproviderappsettings(completionhandler:)?changes=latest_minor)